#! /bin/sh /usr/share/dpatch/dpatch-run
## varrundir.dpatch by Mark Purcell <msp@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Move ASTVARRUNDIR to /var/run/asterisk/

@DPATCH@
diff -urNad asterisk-1.4.2~dfsg~/Makefile asterisk-1.4.2~dfsg/Makefile
--- asterisk-1.4.2~dfsg~/Makefile	2007-03-16 01:41:00.000000000 +0000
+++ asterisk-1.4.2~dfsg/Makefile	2007-03-24 18:28:56.000000000 +0000
@@ -113,7 +113,7 @@
   ASTSBINDIR=$(sbindir)
   ASTSPOOLDIR=$(localstatedir)/spool/asterisk
   ASTLOGDIR=$(localstatedir)/log/asterisk
-  ASTVARRUNDIR=$(localstatedir)/run
+  ASTVARRUNDIR=$(localstatedir)/run/asterisk
   ASTMANDIR=$(mandir)
 ifeq ($(OSARCH),FreeBSD)
   ASTVARLIBDIR=$(prefix)/share/asterisk
